File: AndroidShimmerLayout/app/src/main/java/com/samset/androidshimmerlayout/SecondActivity.java
package com.samset.androidshimmerlayout;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.LinearLayout;
import android.widget.TextView;

import com.samset.samsetshimmerlayout.placeholder.ShimmerFrameLayoutFb;

public class SecondActivity extends AppCompatActivity {
    private ShimmerFrameLayoutFb layoutFb;
    private LinearLayout linearLayout;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);
        layoutFb = findViewById(R.id.shimmer_view_container);
        linearLayout = findViewById(R.id.settings_container);
        layoutFb.startShimmerAnimation();

        linearLayout.postDelayed(new Runnable() {
            @Override
            public void run() {
                layoutFb.stopShimmerAnimation();
            }
        }, 10000);


    }


}

File: app/src/main/java/com/samset/androidshimmerlayout/SecondActivity.kt
package com.samset.androidshimmerlayout

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_second.*

class SecondActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)

        shimmer_view_container?.startShimmerAnimation()
        settings_container?.postDelayed({ shimmer_view_container?.stopShimmerAnimation() }, 1500)


    }


}
